src/main/interface/cli.c

```diff
@@ -3811,7 +3811,7 @@ static void cliVersion(char *cmdline)
 {
     UNUSED(cmdline);
 
-    cliPrintf("# %s / %s (%s) %s %s / %s (%s) MSP API: %s",
+    cliPrintLinef("# %s / %s (%s) %s %s / %s (%s) MSP API: %s",
```

Comment:
This is part of a different and already merged pull request. Please rebase to fix this.